src/main/java/com/teamcautionrobotics/robot/Robot.java

```diff
@@ -25,131 +26,154 @@
+     * teleoperated and test.
+     *
+     * <p>
+     * This runs after the mode specific periodic functions, but before LiveWindow
+     * and SmartDashboard integrated updating.
+     */
+    @Override
+    public void robotPeriodic() {
+        armPower = .5 + .5 * manipulator.getAxis(Axis.LEFT_TRIGGER);
+        velcroHatch.rotate(armPower);
+
+        // if B is pressed, deploy hatch pneumatics
+
+        driveLeftCommand = driverLeft.getY();
+        driveRightCommand = driverRight.getY();
+
+        aimingLights.set(driverLeft.getRawButton(2));
+
+        if (usingVelcroHatch) {
+            if (manipulator.getButton(Button.B)) {
+                velcroHatch.deploy(manipulator.getButton(Button.B));
+                deployButtonPressed = true;
+            } else {
+                if (deployButtonPressed) {
+                    deployButtonPressed = false;
     
-    driveLeftCommand = driverLeft.getY();
-    driveRightCommand = driverRight.getY();
+                    timer.reset();
+                    timer.start();
+                }
+                velcroHatch.deploy(false);
+            }
+            // Counts for .25 of a second
+            if (timer.get() > 0 && timer.get() < 0.25) {
+                driveLeftCommand = -1;
+                driveRightCommand = -1;
+            }
+        } else {
+            if (manipulator.getButton(Button.A) != reacherButtonPressed) {
+                reacherButtonPressed = manipulator.getButton(Button.A);
+                expanderHatch.reach(reacherButtonPressed);
+            }
+
+            if (manipulator.getButton(Button.B) != grabberButtonPressed) {
+                grabberButtonPressed = manipulator.getButton(Button.B);
+                expanderHatch.grab(grabberButtonPressed);
+            }
+        }
+        
+        driveBase.drive(driveLeftCommand, driveRightCommand);
+    }
 
-    if (manipulator.getButton(Button.B))
-    {
-      hatch.deploy(manipulator.getButton(Button.B));
-      deployButtonPressed = true;
+    /**
+     * This autonomous (along with the chooser code above) shows how to select
+     * between different autonomous modes using the dashboard. The sendable chooser
+     * code works with the Java SmartDashboard. If you prefer the LabVIEW Dashboard,
+     * remove all of the chooser code and uncomment the getString line to get the
+     * auto name from the text box below the Gyro
+     *
+     * <p>
+     * You can add additional auto modes by adding additional comparisons to the
+     * switch structure below with additional strings. If using the SendableChooser
+     * make sure to add them to the chooser code above as well.
+     */
+    @Override
+    public void autonomousInit() {
+        m_autoSelected = m_chooser.getSelected();
+        // m_autoSelected = SmartDashboard.getString("Auto Selector", kDefaultAuto);
+        System.out.println("Auto selected: " + m_autoSelected);
```

Comment:
Why is this here?

Reply:
Same reason: branched from an out-of-date master.